cmd/api/api.go

```diff
@@ -73,63 +69,46 @@ var (
 		},
 		cli.StringFlag{
 			Name:        cmd.FlagHTTPAddr,
-			EnvVar:      cmd.EnvHTTPAddr,
 			Usage:       "The address SourcePods API runs on",
 			Value:       ":3020",
 			Destination: &apiConfig.HTTPAddr,
 		},
 		cli.StringFlag{
 			Name:        cmd.FlagHTTPPrivateAddr,
-			EnvVar:      cmd.EnvHTTPPrivateAddr,
 			Usage:       "The address SourcePods runs a http server only for internal access",
 			Value:       ":3021",
 			Destination: &apiConfig.HTTPPrivateAddr,
 		},
 		cli.BoolFlag{
 			Name:        cmd.FlagLogJSON,
-			EnvVar:      cmd.EnvLogJSON,
 			Usage:       "The logger will log json lines",
 			Destination: &apiConfig.LogJSON,
 		},
 		cli.StringFlag{
 			Name:        cmd.FlagLogLevel,
-			EnvVar:      cmd.EnvLogLevel,
 			Usage:       "The log level to filter logs with before printing",
 			Value:       "info",
 			Destination: &apiConfig.LogLevel,
 		},
-		cli.StringFlag{
-			Name:        cmd.FlagSecret,
-			EnvVar:      cmd.EnvSecret,
-			Usage:       "This secret is going to be used to generate cookies",
-			Destination: &apiConfig.Secret,
-		},
 		cli.StringFlag{
 			Name:        cmd.FlagStorageGRPCURL,
-			EnvVar:      cmd.EnvStorageGRPCURL,
 			Usage:       "The storage's gprc url to connect with",
 			Destination: &apiConfig.StorageGRPCURL,
 		},
 		cli.StringFlag{
 			Name:        cmd.FlagStorageHTTPURL,
-			EnvVar:      cmd.EnvStorageHTTPURL,
 			Usage:       "The storage's http url to proxy to",
 			Destination: &apiConfig.StorageHTTPURL,
 		},
 		cli.StringFlag{
 			Name:        cmd.FlagTracingURL,
-			EnvVar:      cmd.EnvTracingURL,
 			Usage:       "The url to send spans for tracing to",
 			Destination: &apiConfig.TracingURL,
 		},
 	}
 )
 
 func apiAction(c *cli.Context) error {
-	if apiConfig.Secret == "" {
```

Comment:
isn't this used for session cookies? :thinking:

Reply:
That's what I thought myself, but it wasn't passed anywhere, so I simply removed it.